index.css

```diff
@@ -549,9 +566,11 @@ body {
 /** Skills */
 .skills {
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-direction: row;
   gap: 24px;
+  margin-top: 120px;
+  justify-content: space-around;
+  flex-wrap: wrap;
```

Comment:
- [ ] Kindly make sure the list of technologies are aligned correctly as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206719963-bea93a30-c322-4c78-baea-9e139328f6d0.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206719988-5203118c-e0ba-4b9d-bb5f-0da58e4c0ae1.png)